waspc/data/Generator/templates/Dockerfile

```diff
@@ -26,7 +26,7 @@ COPY server/ ./server/
 RUN cd server && npm install
 {=# usingPrisma =}
 COPY db/schema.prisma ./db/
-RUN cd server && {= serverPrismaClientOutputDirEnv =} npx prisma generate --schema='{= dbSchemaFileFromServerDir =}'
+RUN cd server && npx prisma generate --schema='{= dbSchemaFileFromServerDir =}'
```

Comment:
Since we again rely on the implicit/default client generation location, the environment variable is unnecessary. I've removed all mentions of it.

Reply:
As mentioned in the other comment, I wouldn't document this. It's Prisma's default and expected behavior.

I does feel like we should write it somewhere, but I think we're just biased due to having a custom setup for all this time. If we had it working like this from the beginning, I don't think we would have thought twice about it.